crates/compiler/test_mono/src/tests.rs

```diff
@@ -2505,7 +2573,7 @@ fn issue_4772_weakened_monomorphic_destructure() {
 
         expect
             result = getNumber
-            result == Ok {val : -1234i64, input : []}
+            result == Ok {{val : -1234i64, input : []}}
         "#
     )
 }
```

Comment:
Same here, though at least in the ticket it seems that the relevant part of the test here is
```
{ result, rest } = ... something that relies on synthesized type of result ...
```

which still seems to hold with any `Decoder.DecoderFormatting` implementation.

Reply:
I think this one is ok